Arrowgene.Ddon.GameServer/Characters/ExpManager.cs

```diff
@@ -707,8 +707,9 @@ public uint CalculateExpBonus(CharacterCommon characterCommon, uint baseExpAmoun
 
         private uint PartyLevelRange(PartyGroup party)
         {
-            uint maxLevel = party.Leader.Client.Character.ActiveCharacterJobData.Lv;
-            uint minLevel = party.Leader.Client.Character.ActiveCharacterJobData.Lv;
+            var firstMember = party.Clients.First();
```

Comment:
This is for a party that continues to kill while the leader left or disconnected?